docs/man_pages/project/testing/debug-ios.md

```diff
@@ -1,5 +1,7 @@
 debug ios
 ==========
+All you need to do to start debugging your app is to execute `tns debug ios`. The NativeScript CLI will build, deploy, start your app, start Chrome DevTools and attach the debugger.
```

Comment:
Do we start Chrome DevTools for iOS?
Also this should be under the `Usage|Synopsis` part of the help. We print the table of usage first in all of the help content.